personal/cezarypukownik/reports/Cwiczenia2.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "Cwiczenia 2"
+output: html_document
+author: 'Cezary Pukownik'
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(knitr)
+library(sf)
+library(raster)
+library(spData)
+library(dplyr)
+library(ggplot2)
+```
+
+# 1. Wyświetlcie mapę krajów Europejskich wg liczby ludności.
+
+```{r}
+world %>%
+  filter(continent == "Europe") %>%
+  select(pop) %>%
+  ggplot(aes(fill=pop)) +
+    geom_sf() +
+    labs(title='Populacja w krajach europejskich') +
+    theme_classic() 
+```
+
+
+
+# 2. Który kraj posiada najmniejszą liczbę ludności i ile wynosi?
+
+
+Najmniejszą liczbę ludności ma Grenlandia.
+
+```{r}
+world %>%
+  arrange(pop) %>%
+  select(name_long,pop) %>%
+  top_n(1, wt=-pop) %>% 
+  st_set_geometry(NULL) %>% 
+  kable()
```

Comment:
Nice touch. Też lubię kable. Skoro jesteś bardziej zaawansowany w tej materii, to polecam też pakiet pander ;)

Reply:
Dzięki! :D kable się ładnie do html kompiluje i jest szybkie w użyciu,  a pander zobaczę.